SinaiStCatherine/SinaiEt005.xml

```diff
@@ -546,7 +549,9 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                       <desc>Untrained and irregular hand, of somewhat later time than <ref target="#h1">Hand 1</ref>.  
                         There is a change of ink (and pen?) between <ref target="#p2_i1"/> and <ref target="#p2_i2"/>, 
                       but it cannot be excluded that the hand is the same. Angular, square letter shapes, somewhat resembling 
-                      the hand of <ref type="mss" corresp="BAVet21"/>. The word for God is written in two units.</desc>
+                      the hand of <ref type="mss" corresp="BAVet21"/>. The word for God is mostly written in two units, 
```

Comment:
I find the use of "word" and "unit" a bit unusual here (it is clear because I know what is meant, but theoretically a "unit" could also be a character). Maybe just "‘God’ is written in two words"?

Reply:
changed